Cargo.lock

```diff
@@ -585,7 +591,7 @@ checksum = "f23ff5ef2b80d608d61efee834934d862cd92461afc0560dedf493e4c033738b"
 dependencies = [
  "hermit-abi 0.3.9",
  "libc",
- "windows-sys",
+ "windows-sys 0.52.0",
```

Comment:
wait why do we have windows-sys?

Reply:
I was confused about that too but I looked into it and it's now required by is-terminal. Maybe we replace is-terminal with a custom implementation that doesn't use imports?

Reply:
o huh thats weird, i think we can leave it in there for now but we should def make an issue to get rid of it later

Reply:
Sounds good. I made #567